packages/fxa-settings/src/pages/InlineRecoveryKeySetup/container.tsx

```diff
@@ -112,7 +117,8 @@ export const InlineRecoveryKeySetupContainer = (_: RouteComponentProps) => {
     !unwrapBKey ||
     !email ||
     !emailForAuth ||
```

Comment:
Is `!emailForAuth` duplicated here?

Reply:
Thanks for the catch. Weird that our linter doesn't complain about this, lol.